aemcmc/negative_binomial_horseshoe_gibbs.py

```diff
@@ -55,12 +96,27 @@ def horseshoe_nbinom_gibbs(
     local_shrinkage_init: TensorVariable,
     global_shrinkage_init: TensorVariable,
     n_samples: TensorVariable,
-):
+) -> Callable:
     r"""
-    Compose a symbolic graph that describes the gibbs sampler of the negative
+    Build a symbolic graph that describes the gibbs sampler of the negative
     binomial regression with a HorseShoe prior on the regression coefficients.
 
-    The implemenation follows the sampler desribed in [1].
+    The implementation follows the sampler described in [1]. It is designed to
+    sample efficiently from the following negative binomial regression model:
+
+    .. math::
+
+        \begin{align*}
+            y_i &\sim \operatorname{NegativeBinomial}\left(\pi_i, h\right)\\
+            h &\sim \pi_h(h) \mathrm{d}h\\
+            \pi_i &= \frac{\exp(\psi_i)}{1 + \exp(\psi_i)}\\
+            \psi_i &= x^T \beta\\
+            \beta_j &\sim \operatorname{Normal}(0, \lambda_j^2\;\tau^2\;\sigma^2)\\
+            \sigma^2 &\sim \sigma^{-2} \mathrm{d} \sigma\\
+            \lambda_j^2 &\sim \operatorname{HalfCauchy}(0, 1)\\
```

Comment:
See comment above regarding the representation of these shrinkage parameters in the model spec.